server/config/config.go

```diff
@@ -802,6 +803,9 @@ func (c *ScheduleConfig) adjust(meta *configMetaData) error {
 	if !meta.IsDefined("enable-joint-consensus") {
 		c.EnableJointConsensus = defaultEnableJointConsensus
 	}
+	if !meta.IsDefined("enable-cross-table-merge") {
+		c.EnableCrossTableMerge = defaultEnableCrossTableMerge
```

Comment:
Can we let the merge is enabled if an upgrade from 4.0?

Reply:
It depends, do you think it's better?

Reply:
I think some paraments should be adjusted to a better default value, even it is upgraded from old version. such as `tolerance-ratio`

Reply:
make sense, but maybe in another PR?